spec/factories/podcasts.rb

```diff
@@ -1,15 +1,15 @@
 FactoryBot.define do
   factory :podcast do
-    name { "MyString" }
-    location { "MyString" }
-    twitter { "MyString" }
-    patreon { "MyString" }
-    instagram { "MyString" }
-    facebook { "MyString" }
-    description { "MyString" }
+    name { Faker::Name.unique.name  }
+    location { Faker::Address.city }
+    twitter { Faker::Hacker.noun  }
+    patreon { Faker::Hacker.noun  }
+    instagram { Faker::Hacker.noun  }
+    facebook { Faker::Hacker.noun  }
+    description {Faker::Movies::HitchhikersGuideToTheGalaxy.marvin_quote }
     content { false }
-    spotify_uri { "MyString" }
-    photo { "MyString" }
+    spotify_uri { Faker::Internet.domain_name }
+    photo { Faker::Placeholdit.image }
     producer { "" }
```

Comment:
Since this is a related faker object am I missing something here?

Reply:
Im loving the fakers
![](https://media.giphy.com/media/JREHb5iIDHyog/giphy.gif)